etc/kayobe/stackhpc-ipa-images.yml

```
@@ -0,0 +1,20 @@
+---
+#################################
+# Release train IPA image sources
+
+# Whether to use Release Train IPA images for Bifrost
+stackhpc_ipa_image_bifrost_enabled: true
+
+# Whether to use Release Train IPA images for Overcloud Ironic
+stackhpc_ipa_image_overcloud_enabled: true
+
+# The IPA image source, defined by os_distribution,
+# os_release, and the current stable version.
+stackhpc_ipa_image_url: "{{ stackhpc_release_pulp_content_url }}/ipa/\
```

If you change the artifact type:
```suggestion
stackhpc_ipa_image_url: "{{ stackhpc_release_pulp_content_url }}/ipa-images/\
```